src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataResource.java

```diff
@@ -185,4 +185,12 @@ ResponseEntity updateCaseDataValue(@PathVariable UUID caseUUID, @PathVariable St
         CaseData caseData = caseDataService.getCaseDataByReference(decodedRef);
         return ResponseEntity.ok(GetCaseResponse.from(caseData, true));
     }
+
+    @Authorised(accessLevel = AccessLevel.READ)
```

Comment:
I don't think we need to restrict access to this method. It will just make it a tiny bit slower.